RescheduleAppointments/README.md

```diff
@@ -25,7 +27,7 @@ We recommend taking the [Quick Start: Lightning Web Components](https://trailhea
     Tip: Make sure that the Dispatched status is not considered a pinned status. From Field Service Settings, go to Scheduling and deselect the Dispatched status from the list of statuses that are considered as pinned, or unmovable, for scheduling.
 
 3. Configure the LWC:
-    1. Open the project in Visual Studio Code. 
+    1. Open the RescheduleAppointments folder in Visual Studio Code. 
```

Comment:
Similar to the readme in the other folders let's format folder names as code, so use `RescheduleAppointments`